spec/abstractops.html

```diff
@@ -1027,32 +1027,26 @@ <h1>ISO 8601 grammar</h1>
           `.`
           `_`
 
-      TZChar :
+      TZChar[Legacy] :
           Alpha
           `.`
           `-`
           `_`
+          [+Legacy] `+`
+          [+Legacy] DecimalDigit
 
-      TimeZoneIANANameComponent :
-          TZLeadingChar TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? TZChar? but not one of `.` or `..`
+      TimeZoneIANANameComponent[Legacy] :
+          TZLeadingChar TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? TZChar[?Legacy]? but not one of `.` or `..`
 
-      TimeZoneIANANameTail :
-          TimeZoneIANANameComponent
-          TimeZoneIANANameComponent `/` TimeZoneIANANameTail
+      TimeZoneIANANameTail[Legacy] :
+          TimeZoneIANANameComponent[?Legacy]
+          TimeZoneIANANameComponent[?Legacy] `/` TimeZoneIANANameTail[?Legacy]
 
       TimeZoneIANALegacyName :
-          `Etc/GMT0`
-          `GMT0`
-          `GMT-0`
-          `GMT+0`
-          `EST5EDT`
-          `CST6CDT`
-          `MST7MDT`
-          `PST8PDT`
+          TimeZoneIANANameTail[+Legacy] [&gt; but only an ASCII-case-insensitive match for one of `Etc/GMT` ASCIISign UnpaddedHour or *"Etc/GMT0"* or *"GMT0"* or *"GMT-0"* or *"GMT+0"* or *"EST5EDT"* or *"CST6CDT"* or *"MST7MDT"* or *"PST8PDT"*]
```

Comment:
Doesn't this grammar match against code points?
```suggestion
          TimeZoneIANANameTail[+Legacy] [&gt; but only if `etc/gmt` |ASCIISign| |UnpaddedHour| matches StringToCodePoints(the ASCII-lowercase of CodePointsToString(the sequence of code points matched by |TimeZoneIANANameTail|))]
          TimeZoneIANANameTail[+Legacy] [&gt; but only if CodePointsToString(the sequence of code points matched by |TimeZoneIANANameTail|) is an ASCII-case-insensitive match for an element of &laquo; *"Etc/GMT0"*, *"GMT0"*, *"GMT-0"*, *"GMT+0"*, *"EST5EDT"*, *"CST6CDT"*, *"MST7MDT"*, *"PST8PDT"* &raquo;]
```
Which could be simplified if we update [ASCII-lowercase](https://tc39.es/ecma402/#sec-case-sensitivity-and-case-mapping) to support parse nodes and/or sequences of code points:
```
          TimeZoneIANANameTail[+Legacy] [&gt; but only if `etc/gmt` |ASCIISign| |UnpaddedHour| matches the ASCII-lowercase of |TimeZoneIANANameTail|]
          TimeZoneIANANameTail[+Legacy] [&gt; but only if the sequence of code points matched by |TimeZoneIANANameTail| is an ASCII-case-insensitive match for an element of &laquo; `Etc/GMT0`, `GMT0`, `GMT-0`, `GMT+0`, `EST5EDT`, `CST6CDT`, `MST7MDT`, `PST8PDT` &raquo;]
```